app/components/forms/order-form.js

```diff
@@ -0,0 +1,110 @@
+import Ember from 'ember';
+import FormMixin from 'open-event-frontend/mixins/form';
+import { countries } from 'open-event-frontend/utils/dictionary/demography';
+
+const { Component, computed } = Ember;
+
+export default Component.extend(FormMixin, {
+  getValidationRules() {
+    var validationRules = {};
+    return {
+      inline : true,
+      delay  : false,
+      on     : 'blur',
+      fields : {
+        firstname: {
+          for(var i = 1; i <= 3; i++){
```

Comment:
I am extremely sorry for this silly mistake! I'll make sure not to repeat such things again. 
The solution you suggested is working fine! Thank you and apologies.